docs/data/date-pickers/base-concepts/ServerInteraction.tsx

```diff
@@ -0,0 +1,152 @@
+                  value === null
+                    ? 'null'
+                    : value.isValid()
+                    ? value.format('DD/MM/YYYY')
+                    : 'Invalid Date'
+                }`,
+            )
+            .join('\n')}
+        </Typography>
+      </Stack>
+
+      <Divider />
+      <Stack sx={{ minWidth: 180 }}>
+        <Typography>onAccept</Typography>
+        <Divider />
+        <Typography component="pre">
+          {logsFromOnAccept
+            .map(
+              (value) =>
+                `- ${
+                  // eslint-disable-next-line no-nested-ternary
+                  value === null
+                    ? 'null'
+                    : value.isValid()
+                    ? value.format('DD/MM/YYYY')
+                    : 'Invalid Date'
+                }`,
+            )
+            .join('\n')}
+        </Typography>
+      </Stack>
+    </Stack>
+  );
+}
+
+// debounce function
+function debounce(func: (...arg: any) => void, wait = 500) {
+  let timeout: NodeJS.Timeout;
+  function debounced(...args: any) {
+    const later = () => {
+      func(...args);
+    };
+    clearTimeout(timeout);
+    timeout = setTimeout(later, wait);
+  }
+
+  debounced.clear = () => {
+    clearTimeout(timeout);
+  };
+
+  return debounced;
+}
+
+function DateFieldWithAccept(
+  props: DateFieldProps<Dayjs> & { onAccept: (value: Dayjs | null) => void },
+) {
+  const { value: valueProp, onAccept, onChange, ...other } = props;
+
+  const [value, setValue] = useControlled<Dayjs | null>({
+    name: 'FieldAcceptValue',
+    state: 'value',
+    controlled: valueProp,
+    default: null,
+  });
+
+  // Debounced function needs to be memoized to keep the same timeout beween each render.
+  // For the same reason, the `onAccept` needs to be wrapped in useCallback.
+  const deboucedOnAccept = React.useMemo(() => debounce(onAccept, 1000), [onAccept]);
+
+  return (
+    <DateField
+      value={value}
+      onChange={(newValue, context) => {
+        setValue(newValue);
+        deboucedOnAccept(newValue);
+        onChange?.(newValue, context);
+      }}
+      {...other}
+    />
+  );
+}
+
+export default function ServerInteraction() {
+  const [logsFromOnChange, setLogsFromOnChange] = React.useState<(Dayjs | null)[]>(
+    [],
+  );
+  const [logsFromOnAccept, setLogsFromOnAccept] = React.useState<(Dayjs | null)[]>(
+    [],
+  );
+
+  const onAccept = React.useCallback((newValue: Dayjs | null) => {
+    setLogsFromOnAccept((prev) => [newValue, ...prev]);
+  }, []);
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <Stack sx={{ textAlign: 'center', alignItems: 'center' }}>
+        <DateFieldWithAccept
+          onAccept={onAccept}
+          onChange={(newValue) => {
+            setLogsFromOnChange((prev) => [newValue, ...prev]);
+          }}
+          sx={{ width: 150 }}
+          label="debounced field"
+          defaultValue={dayjs(new Date(2020, 10, 5))}
```

Comment:
Have you considered using the same default as in most other demos? 🤔 
```suggestion
          defaultValue={dayjs('2022-04-17')}
```